src/libraries/System.Runtime.InteropServices/tests/LibraryImportGenerator.Tests/CollectionMarshallingFails.cs

```diff
@@ -110,6 +110,7 @@ public static partial void NegateBoolsRef2D_ClearMarshalling(
     public class CollectionMarshallingFails
     {
         [Fact]
+        [SkipOnCI("Allocates enough memory that the OOM killer can kill the process on our Helix machines.")]
         public void UTFStringConversionFailures()
```

Comment:
Would it make sense to name this as something specific to conversion failures due to the long string?